src/components/SearchAndSort/SearchAndSort.test.js

```diff
@@ -321,7 +321,7 @@ describe.skip('SearchAndSort component', () => {
     });
   });
 
-  describe('when layer is edit', () => {
+  /* describe('when layer is edit', () => {
```

Comment:
Can we skip these tests for now instead of commenting them?

Reply:
Sorry, my bad. I forgot to remove the comments while I was playing with tests.